Zukini.API/Steps/ZukiniStepContext.cs

```diff
@@ -0,0 +1,68 @@
+﻿using BoDi;
+
+namespace Zukini.API.Steps
+{
+    /// <summary>
+    /// The context class allows users to specify their own data for their step definition classes
+    /// The Zukini StepsContext provides the property bucket and test id by default for the ApiSteps and UISteps
+    /// </summary>
+    /// <remarks>
+    ///     There are two copies of ZukiniStepContext, one in Zukini.API.Steps and one in Zukini.UI.Steps
+    ///     this is because there is a bug in .NET Framework versions 4.5.(0-2) - 4.6.1 that causes a compile error when
+    ///     other projects attemptes to inherit from ApiSteps or UISteps
+    ///     and when ZukiniStepContext is defined in Zukini.Steps
+    ///     Error CS0012  The type 'ZukiniStepContext' is defined in an assembly that is not referenced.You must add a reference to assembly 'Zukini, Version=1.2.5.0, Culture=neutral, PublicKeyToken=null'.	UnitTestProject1 C:\QA\Zukini\UnitTestProject1\UnitTest1.cs  8	Active
+    /// </remarks>
```

Comment:
I still get the error despite the separation here. Also, I thought targeting the .NET 4.6.2 framework resolved this? Is that not the case? I can see that the target version changed in the projects but the issue still persists.

Reply:
Also, Attempts is misspelled (attemptes)

Reply:
targeting the .NET 4.6.2 framework only resolves the issue if the ZukiniStepContext is in Zukini project and not duplicated in Zukini.API and Zukini.UI. That would not help users that target a version < 4.6.2 where the only workaround was to duplicate ZukiniStepContext in both API and UI projects.